viewer/config.txt

```diff
@@ -36,6 +36,7 @@ Drawing Port         : 32769
 General Settings:
 Record Logfiles      : false
 Logfile Directory    : 
+Look and Feel        : intellij
```

Comment:
I still think a comment before this line with available theme names would be helpful so they're more discoverable. That would require tackling #86 though, I guess you don't want to do that during this PR?

Also, I wonder if darcula should be default.

Reply:
I looked into #86 but it's not as easy to fix as i thought.

The default can be what ever we want.

Reply:
I just asked the other members of team magma, we think light theme is the default for most applications. Although dark themes are on the rise. @Gama11 feel free to change the default. But we should definitely mention all possible themes in the read-me and change-log.

Reply:
Side note: if you use darklaf `2.0.3` you use the operating system theme as the default.